--- /dev/null	2023-05-31 14:48:24.769960661 +0300
+++ a/arch/arm/boot/dts/sp7021-icognize.dts	2023-06-01 16:49:17.951077217 +0300
@@ -0,0 +1,27 @@
+/*
+ * Linux TPP3 Gen 2 - LTPP3g2 base board definition
+ * Copyright (C) 2020 Tibbo Tech. - http://tibbo.com/tps/
+ * License: GPLv2 or later
+ */
+/dts-v1/;
+
+#include "sp7021-ltpp3g2revD.dtsi"
+
+/ {
+ model = "SP7021 TPPs3 Gen 2 iCOGNIZE";
+ clocks {
+
+ };
+};
+
+&l2sw {  mode = <1>;  };
+&sdio {  status = "disabled";  };
+
+&pctl {
+
+};
+
+/ {
+    soc@B {
+    };
+};
